public/v4/apps/raneto.yml

```diff
@@ -0,0 +1,41 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: linuxserver/raneto:$$cap_raneto_version
+        environment:
+            TZ: $$cap_tz
+            PUID: '1000'
+            GUID: '1000'
+        hostname: $$cap_appname.$$cap_root_domain
+        volumes:
+            - $$cap_appname-config:/config
+        caproverExtra:
+            containerHttpPort: '3000'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_raneto_version
+          label: Raneto Version
+          defaultValue: 0.17.0-ls49
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/linuxserver/raneto/tags
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_tz
+          label: Time Zone
+          defaultValue: Europe/Brussels
+          description: Get yours from https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+          validRegex: /.{1,}/
+    instructions:
+        start: >-
+            The raneto project aims to provide an open-source software alternative to µTorrent.
```

Comment:
"alternative to µTorrent"? Probably a typo?

Reply:
Let me fix it quickly